SoftLayer/CLI/subnet/detail.py

```diff
@@ -45,6 +48,14 @@ def cli(env, identifier, no_vs, no_hardware):
     table.add_row(['usable ips',
                    subnet.get('usableIpAddressCount', formatting.blank())])
 
+    ip_address = subnet.get('ipAddresses')
+
+    ip_table = formatting.KeyValueTable(['ipAddress', 'value'])
```

Comment:
for the ip_table, lets do `id, IP, Note`, and use a regular table (note KeyValue table)